agent/doc/UML/performance_test_design.puml

```diff
@@ -106,3 +135,32 @@ EspressoRunner -> EspressoPerformanceParser
 EspressoPerformanceParser -> PerformanceInspectionService
 @enduml
 
+@startuml performance_event_test_sequence
+TestRunner -> PerformanceInspection : createEventStartInspection()
+activate PerformanceInspection
+PerformanceInspection -> TestRunner : EventStartInspection
+deactivate PerformanceInspection
+TestRunner -> PerformanceTestManagementService : inspect(EventStartInspection)
+PerformanceTestManagementService -> EventTimeInspector : inspect(EventStartInspection)
+activate EventTimeInspector
+EventTimeInspector -> PerformanceTestManagementService : PerformanceInspectionResult
+deactivate EventTimeInspector
+TestRunner -> PerformanceInspection : createEventEndInspection()
+activate PerformanceInspection
+PerformanceInspection -> TestRunner : EventEndInspection
+deactivate PerformanceInspection
+TestRunner -> PerformanceTestManagementService : inspect(EventEndInspection)
+PerformanceTestManagementService -> EventTimeInspector : inspect(EventEndInspection)
+activate EventTimeInspector
+EventTimeInspector -> PerformanceTestManagementService : PerformanceInspectionResult
+deactivate EventTimeInspector
+TestRunner -> PerformanceTestManagementService : tearDown()
```

Comment:
```suggestion
TestRunner -> PerformanceTestManagementService : analyse()
```
Ran, we will add a new method called analyse in test run lifecycle, refer to this PR for detail: https://github.com/microsoft/HydraLab/pull/442

So maybe we can use this method for result analysis going forward.